.github/workflows/tests.yml

```diff
@@ -20,7 +20,7 @@ jobs: # Docs: <https://git.io/JvxXE>
       matrix:
         setup: ['basic', 'lowest']
         illuminate: [default]
-        php: ['7.3', '7.4']
+        php: [7.4', '8.0', '8.1']
```

Comment:
Why PHP `7.3` deleted?

We need package `ocramius/package-versions` (version `2.1.0`) to support PHP 7.4+

No version of this package supports all versions of PHP (from 7.3 to 8.1)

```bash
- Root composer.json requires ocramius/package-versions 2.1.0 -> satisfiable by ocramius/package-versions[2.1.0].
- ocramius/package-versions 2.1.0 requires php ^7.4.7 || ~8.0.0 -> your php version (7.3.5) does not satisfy that requirement.
```